### PR TITLE
use curl in autogen

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
cc @jburel

### CODE REVIEW

**Comment by jburel** [comment]
2 points I noticed while generating the walkthrough:
 * ``OMEROVER=OMERO-DEV-latest `` => default should be ``latest`` i.e. release version
 * in ``centos7/deps_ice3.6.sh`` and ``ubuntu/deps_ice3.6.sh.j2``, replace ``PyPi`` by ``PyPI``

**Comment by aleksandra-tarkowska** [comment]
systemd updated as pointed by @manics in https://github.com/manics/ansible-role-omero-web/blob/master/templates/systemd-system-omero-web-service.j2#L13

**Comment by aleksandra-tarkowska** [comment]
@jburel try now

File: ansible/templates/common/arg_vars.j2 [hunk 1]
```diff
@@ -3,8 +3,12 @@ For convenience in this walkthrough the main OMERO.web configuration options hav
 
 OMERO_USER={{ omero_user }}
 OMEROVER={{ omero_version }}
```
Comment (jburel) [inline ansible/templates/common/arg_vars.j2]: still have 
``OMEROVER=OMERO-DEV-latest`` as the default
no needed when building the doc

**Comment by jburel** [comment]
Thanks 
merging

**Pull request merged**